### ISSUE TITLE
Behaviour in "jq play 1,6" different from announced in the manual for jq 1.6

### ISSUE THREAD

**Issue by oscaretu** [jqlang/jq#2194]
**Describe the bug**
The results obtained when clicking the Run button in page 

https://stedolan.github.io/jq/manual/v1.6/#Identity:.

are not the expected ones. The example says that running (in _jq play 1.6_) the code

```
jq '. as $big \| [$big, $big + 1] \| map(. > 10000000000000000000000000000000)'
```
using as input

```
10000000000000000000000000000001
```

gives the results:

```
[true, false]
```

but those aren't the real results. The real ones are:

```
[
  false,
  false
]
```

**To Reproduce**

You can reproduce going to https://jqplay.org/s/w_W5hg3mb0

The original URL for jqplay is:

https://jqplay.org/jq?q=.%20as%20%24big%20%7C%20%5B%24big%2C%20%24big%20%2B%201%5D%20%7C%20map(.%20%3E%2010000000000000000000000000000000)&j=10000000000000000000000000000001

I think that in the example it should changed the values of input and the one used to compare, to have something like this:

https://jqplay.org/s/ihmde54MIC

that ejecutes

```
. as $big | [$big, $big + 1] | map(. > 1000000000000000)
```

with input `1000000000000000`.

This gives the result:

```
[
  false,
  true
]
```

But if you add an addition `0` to both the comparison and the input, the response changes to

```
[
  false,
  false
]
```

that seems to indicate that comparing big numbers doesn't work properly, as you can see in https://jqplay.org/s/Cmg74fPKy-

**Expected behavior**
I think that the expected values are wrong, but that the example in the actual state doesn't do what it was intented for.

**Environment (please complete the following information):**
 - jq play version 1.6 online: https://jqplay.org/s/w_W5hg3mb0

**Comment by wtlangford** [comment]
This is actually a documentation bug- the behavior described in the 1.6 manual is wrong. Those large numbers are not precisely representable by IEEE754 doubles. The behavior described by the example (that `10000000000000000000000000000001 > 10000000000000000000000000000000`, but `(10000000000000000000000000000001 + 1)` isn't) is true of the current `master` branch of jq. Since 1.6, we've added some minimal support for big numbers. The current limitation is that you can't do any math with them. If you do, they get converted to IEEE754 doubles (this is a limitation that may be relaxed in future versions, but has some interesting implementation detail problems to solve first).

The appropriate fix here would be to move that example out of the online 1.6 documentation, as it does not belong there.

**Comment by itchyny** [comment]
This is the same issue of #2165 and has been fixed by #2171.
